k8s/demo/common/ia/case-api.yaml

```diff
@@ -20,7 +20,7 @@ spec:
     java:
       replicas: 2
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/ia/case-api:pr-577-f53f84ee
+      image: hmctspublic.azurecr.io/ia/case-api:glob:prod-*
```

Comment:
Why it is `glob` part

Reply:
not sure, some tag done by configuration. I've just reverted it

Reply:
this place is to change the "feed" for new versions:
https://github.com/hmcts/cnp-flux-config/pull/6302/files#diff-95ead830f69722f5a7077e821d244dc8L9

on above we usually put:
image: hmctspublic.azurecr.io/ia/case-notifications-api:latest